admin/test/config/browserstack.yml

```diff
@@ -0,0 +1,73 @@
+server: "hub-cloud.browserstack.com"
+
+common_caps:
+  "browserstack.debug": true
+  "browserstack.local": true
+
+browser_caps:
+  sierra_chrome_58:
+    "os": "OS X"
+    "os_version": "Sierra"
+    "browser": "Chrome"
+    "browser_version": "58.0"
+    "resolution": "1024x768"
```

Comment:
I'm just going to make a general comment here.  At 768 pixels wide our stylesheets will only display the middle view (tablet)  and the mobile and desktop views will not be tested.

So, do we actually need to test in other resolutions?  If not, perhaps we should choose a larger width so that the desktop view is tested.